content/pt-br/docs/reference/glossary/app-container.md

```diff
@@ -0,0 +1,20 @@
+---
+título: Contêiner de aplicações
```

Comment:
The front matter keys should not be translated. "title" should be kept as-is, as translating breaks rendering. The value is fine and should be translated.